src/main/kotlin/ch/hsr/dsl/dwrtc/websocket/WebsocketHandler.kt

```diff
@@ -110,6 +115,7 @@ class WebSocketHandler(app: Javalin, private val signallingService: ClientServic
      *
      * Sends the message to the recipient via WebSocket
      */
+    @Synchronized
```

Comment:
If we switch to https://github.com/http4k/http4k this could be solved aswell.

Reply:
Yes, probably.
Can you make an Idea issue for that (switching to http4k)? Just so we don't forget to consider it.